elements/bulbs-lightbox/bulbs-lightbox.scss

```diff
@@ -0,0 +1,36 @@
+bulbs-lightbox {
+  cursor: zoom-in;
+  .bulbs-lightbox-overlay {
+    align-items: center;
+    background: rgba(0, 0, 0, 0.9);
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    justify-content: center;
+    left: 0;
+    opacity: 0;
+    position: fixed;
+    right: 0;
+    top: 0;
+    transform: scale(0.9);
+    transition: transform 0.2s, opacity 0.2s, visibility 0s 0.2s;
+    visibility: hidden;
```

Comment:
looks like this [won't load the img asynchronously](http://stackoverflow.com/questions/578283/will-an-image-load-if-it-has-its-css-set-so-that-visibility-is-hidden), is that intended?

I made a library kinda like this a little while ago - will help with page load if just the preview is loaded (and the larger version onclick)